client/hotrod-client/src/main/java/org/infinispan/client/hotrod/near/NearCacheService.java

```diff
@@ -228,6 +236,7 @@ public void handleCreatedModifiedEvent(ClientCacheEntryCustomEvent<byte[]> e) {
       }
 
       @ClientCacheEntryRemoved
+      @ClientCacheEntryExpired
```

Comment:
2 methods annotated with `@ClientCacheEntryExpired`?

Reply:
One is for eager and the other is for invalidated near cache.

Reply:
I didn't realize EagerNearCacheListener is still there, but it's private and never used, so I'd rather delete it.
There is a reference to it in the code, but that condition in NearCacheService.createListener() is never false.

Reply:
I will add a sub JIRA under https://issues.jboss.org/browse/ISPN-8070 for it.

Reply:
@wburns I would still like to remove `EagerNearCacheListener` in this PR, I don't think we'll get any eager near cache-related PRs any time soon.

Reply:
Sure